src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/collectors/PerformanceAnalyzerMetricsCollector.java

```diff
@@ -34,6 +34,7 @@
     // ScheduledMetricCollectorsExecutor. A muted collector is skipped.
     MUTED
   }
+
```

Comment:
I added the blankline because of failing checkstyleMain task. 

> Task :checkstyleMain
[ant:checkstyle] [ERROR] /Users/partsrut/Desktop/workspace/performance-analyzer-rca/src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/collectors/PerformanceAnalyzerMetricsCollector.java:37: 'VARIABLE_DEF' should be separated from previous statement. [EmptyLineSeparator]
[ant:checkstyle] [ERROR] /Users/partsrut/Desktop/workspace/performance-analyzer-rca/src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/jvm/ThreadList.java:259:103: '||' should be on a new line. [OperatorWrap]

> Task :checkstyleMain FAILED